.github/workflows/api-release.yml

```
@@ -0,0 +1,38 @@
+runs-on: ubuntu-latest
```

actionlint (1.7.4)

unexpected key "runs-on" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)